docs/20_smart-contracts/10_getting-started/10_dune-guide.md

```diff
@@ -2,153 +2,150 @@
+Each platform has their own installation instructions. Click [DUNE's GitHub project](https://github.com/AntelopeIO/DUNE) to view the instructions.
 
-Once you're done, you can run `dune --help` to see a list of all supported commands.
+Run `dune --help` to view a list of all supported commands.
 
 ## Wallets
 
-DUNE handles wallet management for you so that you don't have to. 
+DUNE handles wallet management for you so you don't have to. 
 
 If you need to import a new key into your wallet:
 
 ```shell
 dune --import-dev-key <PRIVATE_KEY>
 ```
 
-## Node Management
+## Node management
 
-Creating a new local EOS blockchain is easy with DUNE.
+DUNE easily creates a new local EOS blockchain.
+
+The following command creates a new node called `NODE_NAME`.  This node initiates with default settings. These default settings configure the node to serve as an API/producer node. You can deploy smart contracts to it and perform tests on it.
 
 ```shell
 dune --start <NODE_NAME>
 ```
 
-The command above creates a new node called `NODE_NAME` and starts it with default settings. 
-The node is configured to serve as an API/producer node that you can deploy smart contracts to, and perform tests on.
-
-> ❔ **Errors**
+> 📝 **In Case of Errors**
 >
 > You may see errors at the end of the node setup process.
-> If you do you can refer to this guide to troubleshoot common errors, or reach out to us on our
+> If you do, refer to this guide to troubleshoot common errors, or reach out to us on our
 > [Telegram channel](https://t.me/antelopedevs) for help.
 
-You can see a list of EOS nodes on your system:
+The following command shows you a list of EOS nodes on your system:
 
 ```shell
 dune --list
 ```
 
-You can also check if your active node's RPC API is live:
+The following command checks if your active node's RPC API is live:
 
 ```shell
 dune -- cleos get info
 ```
 
-To shut down your node:
+The following command shuts down your node:
 
 ```shell
 dune --stop <NODE_NAME>
 ```
 
-To remove a node entirely:
+The following command removes a node:
 
 ```shell
 dune --remove <NODE_NAME>
 ```
 
-
 ### Bootstrapping your environment
 
-There are a few system contracts that your development environment might need to rely on such as:
+There are a few system contracts that your development environment may need to rely on, such as:
 - `eosio.token` for **EOS** token transfers
 - `eosio.msig` for multisig transactions
 - `eosio.system` for system level actions such as resource management
 
-Bootstrapping your local node is easy, once you have an active node running, you can bootstrap it with:
+Bootstrapping your local node is easy. Once you have an active node running, bootstrap it with:
 
 ```shell
 dune --bootstrap-system-full
 ```
 
-
 ## Account management
 
-You deploy contracts on top of accounts, and also use them to interact with your smart contracts. 
+You deploy contracts on top of accounts. Use accounts to interact with your smart contracts. 
```

Comment:
This changes the meaning, please revert.

Reply:
These changes were done to simplify the language for Hemingway.

Reply:
Hemingway is just a tool, it does not mean aligning with it allows tw review to change the meaning of technical documents.

Reply:
I am using Hemingway to meet the requirement for 5th grade level. Therefore most sentences will be shortened or divided into two or more sentences. This type of change is not meant to change the meaning, but to clarify the meaning of the information in smaller chunks in a simpler structure. That structure is noun. verb, object. It does not leave much space for nuance. 

Using the term "them" needs to be explicit - do you mean contracts or accounts?

Can I use the following? "You can deploy contracts on top of accounts. An account with a contract on it is no different from an account you use to interact with smart contracts." - that makes more sense to me.